db/migrate/20160211082126_create_events.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
+      t.string :title
+      t.datetime :time
+      t.string :tickets
+
+      t.timestamps null: false
+    end
```

Comment:
can you also add the foreign key linking an event to a proposal? It should be a non-nullable, indexed integer. See http://edgeguides.rubyonrails.org/active_record_migrations.html#foreign-keys